api/LargestContentfulPaint.json

```diff
@@ -187,6 +187,44 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cross-origin-render-time": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/API/LargestContentfulPaint/renderTime#cross-origin_image_render_time",
+            "spec_url": "https://w3c.github.io/largest-contentful-paint/#dom-largestcontentfulpaint-rendertime",
```

Comment:
Let's omit this spec url, since it seems to be the same for the parent, so it's not specific to this feature:

```suggestion
```